source/guides/maintenance/cpu-performance.rst

```diff
@@ -193,6 +193,53 @@ group:
    After adding a new group, you must log out and log back in for the new group
    to take effect.
 
+Thermal configuration
+=====================
+
+Better thermal control and performance can be archived by providing platform
+specified configuration to thermald.
+
+`Linux DPTF Extract Utility`_ is a companion tool to thermald, This tool can
+make use of "Intel ® Dynamic Platform and Thermal Framework (`Intel DPTF`_)"
+technology, and convert to the thermal_conf.xml configuration format used
+by thermald.
+
+First, make sure thermald version is equal or above 1.9.1:
+
+.. code:: bash
+
+   thermald --version
+
+Otherwise, you need to add an option :command:`--ignore-default-control` to
+thermald:
+
+.. code:: bash
+
+   # edit thermald.servcie
+   vim /usr/lib/systemd/system/thermald.service
+   # append --ignore-default-control option to ExecStart line, like below
+   # ExecStart=/usr/bin/thermald --no-daemon --dbus-enable --ignore-default-control
+
+   # reload
+   sudo systemctl daemon-reload
+
+Then, generate thermal configuration as below:
+
+.. code:: bash
+
+   git clone https://github.com/intel/dptfxtract.git
+   cd dptfxtract
+   sudo acpidump > acpi.out
+   acpixtract -a acpi.out
+   sudo ./dptfxtract *.dat
```

Comment:
When I tested these commands, `sudo ./dptfxtract *.dat` produced the following: 
```
...
No valid tables found
```
Is this an error?

Reply:
DPTF need BIOS support. Most Intel laptop support this feature.
Typical path is: Configuration -> DPTF (Enabled)

Reply:
I can not find such a guide/article to enable DPTF in BIOS. And menu entry might changed from different vendor.

And from the project of dptf for chromimum OS, here is what they says:

> Requirement: DPTF requires corresponding ACPI support in BIOS. Not all
Intel based platforms support DPTF in BIOS. Please contact your BIOS vendor
to see if DPTF is enabled in your system.

https://github.com/intel/dptf

Reply:
Given that the option to enable may vary per the BIOS, I'll let this stand as is. Refinements to this will be addressed in a follow-up GH Issue.